Test: Cannot find module 'grpc' from 'grpc_connection.js'

```
npm t
```

```sh
 FAIL  src/App.test.js
  ● Test suite failed to run

    Cannot find module 'grpc' from 'grpc_connection.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:179:17)
      at Object.<anonymous> (node_modules/@firebase/firestore/dist/cjs/src/platform_node/grpc_connection.js:29:12)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.045s
Ran all test suites.
```

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "deploy": "gh-pages -d build",
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "test": "react-scripts test --env=jsdom --browser",
     "eject": "react-scripts eject"
   }
 }

--- a/src/App.test.js
+++ b/src/App.test.js
@@ -1,8 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import App from './components';
 
 it('renders without crashing', () => {
   const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
+  ReactDOM.render(
+    <MuiThemeProvider>
+      <App />
+    </MuiThemeProvider>,
+    div
+  );
 });

